bson/codec_options.py

```diff
@@ -198,13 +199,25 @@ def __eq__(self, other: Any) -> Any:
         )
 
 
+class DatetimeConversionOpts(str, enum.Enum):
+    DATETIME = "datetime"
+    DATETIME_CLAMP = "datetime_clamp"
+    DATETIME_MS = "datetime_ms"
+
+    def __str__(self):
+        return f"'{self.value}'"
+
+    __repr__ = __str__
```

Comment:
I don't think `__repr__/__str__` is needed for enums is it?

Reply:
I included it for `__repr__` in `CodecOptions`, since the behavior of enums is different and requires referencing `.value` otherwise. If we don't include it, `__repr__` and `__str__` look like this:

```
In [127]: DatetimeConversionOpts.DATETIME
Out[127]: <DatetimeConversionOpts.DATETIME: 'datetime'>

In [128]: str(Out[127])
Out[128]: 'DatetimeConversionOpts.DATETIME'
```

Reply:
I see. I think it's preferable to have `DatetimeConversionOpts.DATETIME` in the repr/str output for CodecOptions.